.travis.yml

```diff
@@ -1,3 +1,8 @@
+# Get a less shallow clone; needed to get the commit count since
+# the previous release branch point right.
+git:
+  depth: 3000
```

Comment:
You could use `false` to disable the depth restriction.

Reply:
Yeah I know, I chose not to because I only need the commits till the last branch point. Getting all 25,000 of them can be quite slow.

Reply:
3,000 should be enough for any minor release period I think.